### PR TITLE
Remove any references to rubocop-todo.yml

### PR DESCRIPTION
**Pull request by ptarjan**
As discussed in https://github.com/bbatsov/rubocop/pull/3013

* [x] Wrote [good commit messages][1].
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it)
* [x] Squashed related commits together.
* [ ] Added tests.
* [x] Added an entry to the [Changelog](CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../CONTRIBUTING.md#changelog-entry-format).
* [x] All tests are passing.
* [x] The new code doesn't generate RuboCop offenses.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.

[1]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

### CODE REVIEW

**Comment by bbatsov** [comment]
Capitalise the commit message and mention this change in the changelog.

**Comment by ptarjan** [comment]
@bbatsov done, thanks

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * [#2997](https://github.com/bbatsov/rubocop/pull/2997): `Performance/CaseWhenSplat` can now identify multiple offenses in the same branch and offenses that do not occur as the first argument. ([@rrosenblum][])
 * [#2928](https://github.com/bbatsov/rubocop/issues/2928): `Style/NestedParenthesizedCalls` cop can auto-correct. ([@drenmi][])
 * `Style/RaiseArgs` cop can auto-correct. ([@drenmi][])
+* [#3025](https://github.com/bbatsov/rubocop/pull/3025): Removed deprecation warnings for `rubocop-todo.yml`. ([@ptarjan][])
```
Comment (bbatsov) [inline CHANGELOG.md]: This should be in changes, not new features.

**Comment by bbatsov** [comment]
👍

**Pull request merged**